### DISCUSSION TITLE
Connect Safe with WalletConnect

### DISCUSSION THREAD

**Discussion by saulodigital** [WalletConnect/walletconnect-monorepo#3417] [Developer Support]
I'm trying to connect Safe with WalletConnect via an actual Safe App but there's an issue (screenshots below).

Is there any ETA when this issue will be solved? Thank you!

![Screenshot of error connecting with Safe](https://i.imgur.com/llmTTQp.png)

![Safe App of WalletConnect](https://i.imgur.com/4IYbjSg.png)

**Comment by glitch-txs** [comment]
Thanks for reporting! We are aware of this issue and we're currently working on it 🙏 

No ETA at the moment* we'll anounce once it's fixed
